### PR TITLE
New article: Delete rackspace email mailbox

### PR DESCRIPTION
**Pull request by DonNateR**
From PR #2106

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-9db5eb8075/how-to/)

**Comment by kmsholcomb** [comment]
Hi @DonNateR. Edits are attached. I edited both articles to be as similar as possible, so some changes might seem random! If you have a question, see the other article. Thanks!

[Delete a Rackspace Email mailbox_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1146005/Delete.a.Rackspace.Email.mailbox_editsKH.pdf)
[Delete a Rackspace Email mailbox_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1146006/Delete.a.Rackspace.Email.mailbox_editsKH.docx)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-942e57e872/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-df2b12f700/how-to/)

File: content/rackspace-email/delete-a-rackspace-email-mailbox.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+4. Check the box to the left of the mailbox you wish to delete.
+
+    **Warning:** A deleted mailbox can be recovered for up to 14 days. For instructions, see [Recover a deleted mailbox](/how-to/recover-a-deleted-rackspace-email-mailbox/).
+
+5. Expand the **Select Action** menu at the bottom of the mailbox list, and select **Delete** mailboxes.
+
+   <img src="{% asset_path rackspace-email/delete-a-rackspace-email-mailbox/delete-rse-box-sc2.png %}" />
+
+6. Read the information in the popup box that appears, and if there is no conflict, click **Delete 1 mailbox**.
```
Comment (catlook) [inline content/rackspace-email/delete-a-rackspace-email-mailbox.md]: ... and, if there is no ...

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e3355731e6/how-to/).

**Pull request merged**